README.md

```diff
@@ -1,3 +1,5 @@
 # glibc-bootstrap
 
-TODO
+Homebrew bottles on Linux must be linked to a version of `glibc` than the equivalent or newer than the version used when they were compiled in CI.  If the host `glibc` is too old, we install Homebrew's `glibc` first so that all bottles will use the Homebrew version of `glibc` instead of the host `glibc`.
```

Comment:
```suggestion
Homebrew bottles on Linux must be linked to a version of `glibc` ??? than the equivalent or newer than the version used when they were compiled in CI.  If the host `glibc` is too old, we install Homebrew's `glibc` first so that all bottles will use the Homebrew version of `glibc` instead of the host `glibc`.
```

I think a word is missing here but not 100% sure what it's supposed to be